resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/ExecutorPodsWatchSnapshotSource.scala

```diff
@@ -86,12 +86,20 @@ class ExecutorPodsWatchSnapshotSource(
     }
 
     override def onClose(e: WatcherException): Unit = {
-      logWarning("Kubernetes client has been closed (this is expected if the application is" +
-        " shutting down.)", e)
+      if (SparkContext.getActive.map(_.isStopped).getOrElse(true)) {
```

Comment:
Should we use false as default?

Reply:
Do you mean `getOrElse(false)`?

Here, when `SparkContext.getActive` is `None`, it should be considered as `stopped`.

Reply:
Oh okay, if it is only case `getActive` return None, then it is fine.

Reply:
Thank you!